releasenotes/releasenotes_V0.7.2.md

```diff
@@ -0,0 +1,13 @@
+# Release Notes 0.7.2
+
```

Comment:
should we also list that we have implemented a breaking change regarding the `keptnapi` / `keptnapiv1` topic, and how developers should deal with that migration path?

Reply:
I will mention that in the release notes and add a detailed section in the readme